crates/holochain/tests/tests/agent_activity.rs

```diff
@@ -22,6 +22,10 @@ async fn get_agent_activity() {
     let alice_cell = cells.first().unwrap();
     let bob_cell = cells.last().unwrap();
 
+    conductor_batch[0]
```

Comment:
@neonphog This test was not flaky, it was broken. Without the full storage arc, it never passed for me.